source/src/ports/POSIX/sample_application/sampleapplication.c

```diff
@@ -21,56 +22,329 @@
 #include "ethlinkcbs.h"
 #endif
 
-#define DEMO_APP_INPUT_ASSEMBLY_NUM 100                  // 0x064
-#define DEMO_APP_OUTPUT_ASSEMBLY_NUM 150                 // 0x096
-#define DEMO_APP_CONFIG_ASSEMBLY_NUM 151                 // 0x097
-#define DEMO_APP_HEARTBEAT_INPUT_ONLY_ASSEMBLY_NUM 152   // 0x098
-#define DEMO_APP_HEARTBEAT_LISTEN_ONLY_ASSEMBLY_NUM 153  // 0x099
-#define DEMO_APP_EXPLICT_ASSEMBLY_NUM 107                // 0x6B
+#include <unistd.h>
+
+#define INTERROLL_DEVICE_INPUT_ASSEMBLY_NUM     101
+#define INTERROLL_DEVICE_OUTPUT_ASSEMBLY_NUM    100
+#define INTERROLL_DEVICE_CONFIG_ASSEMBLY_NUM      1
+
+// The values below are incorrect and need to updated when we 
+// start using them.
+#define INTERROLL_DEVICE_HEARTBEAT_INPUT_ONLY_ASSEMBLY_NUM 152   // 0x098
+#define INTERROLL_DEVICE_HEARTBEAT_LISTEN_ONLY_ASSEMBLY_NUM 153  // 0x099
+#define INTERROLL_DEVICE_EXPLICT_ASSEMBLY_NUM 107                // 0x6B
+
+#define MAX_ZONES 4
+#define FIRST_ZONE 0
+#define LAST_ZONE 3
+#define ZONE_0 0
+#define ZONE_1 1
+#define ZONE_2 2
+#define ZONE_3 3
+
+// Arbitrary package transition time, will likely needed to be tuned over time.
+#define PACKAGE_TRANSITION_TIME 600000
```

Comment:
what are the units here?

Reply:
Its in micro-seconds. I'll update the macro definition to reflect that.